pvacseq/lib/parse_output.py

```diff
@@ -188,7 +194,6 @@ def match_wildtype_and_mutant_entry_for_inframe_indel(result, mt_position, wt_re
             #We then check if the alternate best match epitope has more matching amino acids than the baseline best match epitope
             #If it does, we pick it as the best match
             if consecutive_matches_from_right > best_match_count:
```

Comment:
Does any of this left/right stuff even matter anymore, since we are no longer matching from the ends?